src/texas_host/core.clj

```diff
@@ -45,4 +45,4 @@
 (defn -main
   "I don't do a whole lot ... yet."
   [& args]
-  (println (str "John got the following cards " (deal-hand "john"))))
+  (println (str "John got the following cards " (deal-hand "bert"))))
```

Comment:
:) It was a mishap. It probably was the first thing I changed in the code a way back.